apps/client/src/components/components.test.js

```diff
@@ -0,0 +1,148 @@
+import "jest-styled-components";
+
+import React from "react";
+
+import matchMedia from "../__mocks__/matchMedia";
+import { EDIT_BUTTON } from "../utils/test-ids";
+import { act, cleanup, fireEvent, render } from "../utils/test-utils";
+import AddressLine from "./AddressLine";
+import ContactSentence from "./ContactSentence";
+import PrintDetails from "./PrintDetails";
+import QuestionAnswer from "./QuestionAnswer";
+import RegisterLookupSummary from "./RegisterLookupSummary";
+
+Object.defineProperty(window, "matchMedia", matchMedia);
+
+jest.mock("react-router-dom", () => ({
+  ...jest.requireActual("react-router-dom"),
+  useParams: () => ({}),
+}));
+
+let onClickMock;
+
+afterEach(() => {
+  onClickMock = jest.fn();
+
+  cleanup();
+});
+
+// These tests are meant to test only the most basic components. Advanced components need their own file.
```

Comment:
I understand but this is not very portable. Also you need to do more work when a component grows/shrinks, is moved to `atoms` for example.

Reply:
Done, thanks for the feedback